samples/vlc-sample/simple.def

```diff
@@ -4,7 +4,7 @@ src/*
 Done processing src/libvlc.c
 Done processing src/libvlc.h
 Done processing src/missing.c
-Total errors found: 602
+Total errors found: 600
```

Comment:
Should we be adding new tests instead of just removing old ones?

Reply:
I don't see what new test would be applicable here. We still err on these errors in other testcases.